src/coreclr/tests/helixpublishwitharcade.proj

```diff
@@ -78,7 +78,7 @@
 
     <IncludeDotNetCli>true</IncludeDotNetCli>
     <DotNetCliPackageType>runtime</DotNetCliPackageType>
-    <DotNetCliVersion>5.0.0-preview.5.20227.15</DotNetCliVersion>
+    <DotNetCliVersion>5.0.0-preview.5.20228.8</DotNetCliVersion>
```

Comment:
You should be able to change this to the following to have the Helix SDK pull down the same runtime version as the one we build against.

```suggestion
    <DotNetCliVersion>$(BundledNETCoreAppPackageVersion)</DotNetCliVersion>
```